Frameworks/UIKit/NSTextContainer.mm

```diff
@@ -17,12 +17,19 @@
 #include "Starboard.h"
 
 #include "UIKit/NSParagraphStyle.h"
-#include <CGPathInternal.h>
+#include <CGContextInternal.h>
 
+#include <vector>
+#include <algorithm>
+
+static constexpr CGFloat c_defaultPadding = 5.0f;
 @implementation NSTextContainer {
     CGSize _size;
     NSLayoutManager* _layoutManager;
     StrongId<NSArray> _exclusionPaths;
+    // TODO 1394: Remove for CGD2D
```

Comment:
nit: doesn't matter too much since these'll eventually be removed, but if the counts of these two vectors are expected to stay the same, it's good to put in an enforcement mechanism for that (ie: use one vector, or even an std::map)

I would just add a comment noting that assumption in this case, since we'll eventually remove this